mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
```

Comment:
Inconsistent indentation. The `.then` statement should be indented to match the surrounding code block.